StRoot/StETofUtil/StETofMessageFormat.h

```diff
@@ -340,6 +350,8 @@ namespace gdpbv100 {
          static double   CalcDistanceD(double start, double stop);
 
          bool operator<(const gdpbv100::Message& other) const;
+         bool operator==(const gdpbv100::Message& other) const;
+         bool operator!=(const gdpbv100::Message& other) const;

```

Comment:
You may want to inline these operators as the rest of the methods of the Message type. There could be a good reason why it was done this way...

Reply:
For now, the only reason to keep it this way is consistency with the CBM version of this file which may make future debugging easier.

I can change it if there is an advantage to this.

Reply:
It was just a suggestion. I don't have any objective arguments either way, nor I am going to do any measurements. So, as you see I took the chances and approved. Hopefully, this code will not cause any slowdown in our reconstruction chains (if it is used at all).